certified-connectors/Impower/apiDefinition.swagger.json

```diff
@@ -2216,7 +2126,7 @@
         ]
       }
     },
-    "/pmp-accounting/api/v1/invoices/id": {
+    "/services/pmp-accounting/api/v1/invoices/id": {
```

Comment:
The path template has been changed for the operation. can you please confirm that no existing flows or users will be affected by this change?

Reply:
the 2 legacy operations:

1. upload & get invoice can be found at this dedicated URL
2. what changed since the LAST MASTER release is that this api definition does NOT have a basepath, that is why this prefix is included in the operation path